### PR TITLE
feat(packages): update config for release version v8.4.0+

### PR DESCRIPTION
**Pull request by wuhuizuo**
Start v8.4.0:
- remove tidb-binlog artifacts and files.
  - remove `binlogctl` in `ctl` tiup pkg.
  - not build tidb-binlog from v8.4.0.
  - remove `pump` and `drainer` tiup packages and other binaries from tidb-binlog into offline deloyment packages.
- bump etcdctl version to v3.5.15

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: packages/offline-packages.yaml.tmpl [hunk 1]
```diff
@@ -52,8 +52,83 @@ editions:
       tags:
         - "{{ .Release.version }}-community"
     routers:
-      - description: "Started from v7.6.0 to latest added tiproxy compare with v7.5.0"
-        if: {{ semver.CheckConstraint ">=7.6.0-0" .Release.version }}
+      - description: "Started from v8.4.0"
```
Comment (Benjamin2037) [inline packages/offline-packages.yaml.tmpl:55]: Why here add that much contents also why changed it from v7.6.0 to v8.4.0-0?

Comment (wuhuizuo) [reply packages/offline-packages.yaml.tmpl:55]: @Benjamin2037 
The diff page is not pretty, you can ignore it: it's a new route.

image

**Review by Benjamin2037 — APPROVED** [review]
Rest LGTM

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**